Loop/Managers/NotificationManager.swift

```diff
@@ -161,11 +140,11 @@ extension NotificationManager {
     }
     
     @MainActor
```

Comment:
These are the only in-app failure notifications I left behind -- for bolus/carb specific issues (i.e. not parsing errors or OTP erorrs -- things like "The bolus is too big").

We may decide we don't want these either but I left them for now as they were pretty specific.

Reply:
Things like "the bolus is too big" is something the caregiver ideally should be told at the time of entering the command.

Reply:
Yeah the caregiver app does prevent really large values from being entered (not sure about Nightscout) but doesn't yet have support for honoring the Looper's dosing maximums.